src/components/Header/Header.jsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
This ignore must have been necessary at some point, but looks like it's not needed anymore 🤔